src/lib/LUA/tx_devLUA.cpp

```diff
@@ -707,6 +707,7 @@ static void registerLuaParameters()
       if (isDisconnected)
       {
         config.SetLinkMode(arg);
+devicesTriggerEvent(); // trigger event to let the backpack know the link mode has changed (see devBackpack)
```

Comment:
Should not need this as the devices are notified once the config is committed in the main loop.

Reply:
So you're right - done. :)